free-3.md

```diff
@@ -57,12 +88,22 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Building the language aspects shouldn't be time-consuming in comparison to the
+systems aspects. This is because there's only a limited number of different settings/
+features/functions you can have for lights, but helping the user see digitally what 
```

Comment:
Agreed&mdash;I think modeling the user choices would be rather difficult, but you could limit the actually implemented visualizations to make this project more possible in the limited time we have.